File: WebAPI_ValeTransporte/API_VT/API_VT/Models/Despesa.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace API_VT.Models.Entities
{
    public class Despesa
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [DataType(DataType.Date)]
        [DisplayFormat(DataFormatString = "{dd/MM/yyyy}")]
        public DateTime DataInicial { get; set; }

        [Required]
        [DataType(DataType.Date)]
        [DisplayFormat(DataFormatString = "{dd/MM/yyyy}")]
        public DateTime DataFinal { get; set; }

        [Required]
        public double ValorTotal { get; set; }

        public List<DespesaFuncionario> DespesasFuncionarios { get; set; }
    }
}

File: ValeTransporte/src/app/models/Despesa.ts
import { DespesaFuncionario } from './DespesaFuncionario';

export class Despesa {

    id: number;
    dataInicial: Date;
    dataFinal: Date;
    valorTotal: number;
    despesaFuncionario: DespesaFuncionario;

    constructor() {
        this.id = 0;
        this.dataInicial = null;
        this.dataFinal = null;
        this.valorTotal = 0.0;
    }
}
